app/TBG.py

```diff
@@ -112,12 +113,15 @@ def create_new_game() -> Dict:
         player: Player = Player(request.json['name'])
     except KeyError:
         abort(400, util.error('Name not supplied'))
-    
-    public: bool = False
-    if 'public' in request.json:
-        public = request.json['public']
 
-    game: Game = Game(public)
+    try:
+        if not request.json['game_type'] in GAME_TYPES:
```

Comment:
Change to the more readable `if X not in Y`

Also GAME_TYPES will change to Game.GAME_TYPES since we're adding it to the class.

Reply:
Update: make GAME_TYPES simply ('public', 'private') since this is the only place it's being used in the codebase.

Reply:
What if we want to reference the available game types later?  Wouldn't it make sense to either put it in a class or have a constant that represents the types?

Reply:
Off the top of my head, I don't see any other location we would need to reference all available game types. If we get to a point where it is being used in multiple places, we can pull it out and make it a constant. Until then, I don't think I want to pollute the model with a one-off.

Reply:
Yeah, I can't think of a situation, it just might be useful eventually.  But you're right; who cares right now.  

I'll pop it back in as a tuple and we'll keep chugging.